module5/TaskListApp/TaskListApp/Views/TaskListView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  TaskListView.swift
+//  TaskListApp
+//
+//  Created by Peter Rice on 12/22/23.
+//
+
+import SwiftUI
+
+struct TaskListView: View {
+  @ObservedObject var tasksStore: TasksStore
+  @State var showingAddTaskView: Bool = false
+  
+  var body: some View {
+    NavigationStack {
+      VStack(alignment: .leading) {
+        InlineTaskListView(tasksStore: tasksStore)
+        Spacer()
+        FooterView(showingAddTaskView: $showingAddTaskView)
+      }
+      .navigationTitle(Text("My Tasks"))
+      .navigationDestination(for: Task.self) { task in
```

Comment:
Another option here would be to bind task itself in the detail view. so 
````
navigationDestination(for: Task.self) { $task in
 TaskDetailsView($task)
}
````
then anything you change on the detaiil will be refelected. Your way is fine as well, just wanted to point out the alternative.